hypertrace-metrics-processor/hypertrace-metrics-processor/src/main/java/org/hypertrace/metrics/processor/MetricsEnricher.java

```diff
@@ -0,0 +1,22 @@
+package org.hypertrace.metrics.processor;
+
+import io.opentelemetry.proto.metrics.v1.ResourceMetrics;
+import org.apache.kafka.streams.KeyValue;
+import org.apache.kafka.streams.kstream.Transformer;
+import org.apache.kafka.streams.processor.ProcessorContext;
+
+public class MetricsEnricher
+    implements Transformer<byte[], ResourceMetrics, KeyValue<byte[], ResourceMetrics>> {
+
+  @Override
+  public void init(ProcessorContext context) {}
+
+  @Override
+  public KeyValue<byte[], ResourceMetrics> transform(byte[] key, ResourceMetrics value) {
+    // noop enricher for now
+    return new KeyValue<>(key, value);
```

Comment:
In the first iteration, we will be extracting metrics from traces, and they will already have enriched attributes as labels. We will enhance this later, for now, they are just pass through. If requires, we can check if the metrics contain API and service id, and if not, we can drop them.